app/game.rb

```diff
@@ -39,6 +41,7 @@ def draw
     @map.draw
     @scope.draw
     @bullets.draw
+    @lifes_counter.draw("LIFES: #{@player.lifes}", 10, 10, 0, 1.0, 1.0, Gosu::Color::YELLOW)
```

Comment:
Can we extract this logic into a new class? So that you do `@lifes_counter.draw` and then `@lifes_counter` knows what text to write, the colour, etc. This way we keep `Game` skinny.